docs/_releases/v0.5.38.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: release
+date: 2021-07-14
+tag: v0.5.38
+---
+
+## What's New
+## 🔤 General
+- update go-version in CI doc @navendu-pottekkat (#3582)
+- Updated UI to show SMI version @dhruv0000 (#3580)
+- graphql refactor @kumarabd (#3567)
+- [Meshery] Reworking `/experiment/pattern` endpoints @piyushsingariya (#3542)
+- @vijcheru Called Meshkit Error Handler instead of Logrus @vijcheru (#3387)
+- [Mesheryctl] server_spec_handler (meshkit) @Chadha93 (#3561)
+- [Mesheryctl] session_sync_handler (meshkit) @Chadha93 (#3560)
+- [mesheryctl] fix: Added description for the `system logs` command @muskankhedia (#3284)
+- Sets default value for namespace in management page @sudo-NithishKarthik (#3564)
+- Add settings page path to exportPathMap @sudo-NithishKarthik (#3548)
+- Fix indentation problem in label commentor workflow file @sudo-NithishKarthik (#3552)
+- Issue #3401/Added field token-location in meshery system context view @Revolyssup (#3408)
+- refactor mesheryctl 1 - Add getters and setters to config and refactor start.go @navendu-pottekkat (#3546)
+- use latest version of error codes utility in the err-codes-updater workflow @DelusionalOptimist (#3539)
+- [UI] Pass remote provider page titles verbatim @sudo-NithishKarthik (#3532)
+- Migrated errors for models package #2 @dhruv0000 (#3528)
+- fix for prometheus connectivity bug @kumarabd (#3537)
+- updated Validator.go @Shreyas220 (#3533)
+- mesheryctl features @kumarabd (#3514)
+- fix broken dependency @killbotXD (#3526)
+- creating issue/remind label text @Anita-ihuman (#3523)
+- [mesheryctl] `mesheryctl pattern apply -f <github-url>` @piyushsingariya (#3489)
+- Add icon dependencies for remote provider  @sudo-NithishKarthik (#3519)
+- [UI]add duplicate button @killbotXD (#3524)
+- [Fix]: Re-enable support for patterns, filters and applications in local provider @utkarsh-pro (#3515)
```

Comment:
I think this statement is missing.

Reply:
@ghulamyazdani-bot Here's the reference in case you need to cross verify, https://github.com/meshery/meshery/releases/tag/v0.5.38

Reply:
Yes, But if you do not mind can you investigate this why this change is not getting reflected ?